apps/src/applab/designMode.js

```diff
@@ -553,6 +553,23 @@ designMode.serializeToLevelHtml = function () {
   Applab.levelHtml = serialization;
 };
 
+
+function getUnsafeHtmlReporter(sanitizationTarget) {
+  function reportUnsafeHtml(sanitizationTarget, removed, unsafe, safe) {
```

Comment:
This could just  be:
```
return function (removed, unsafe, safe) {
  ...
}
```
no?

Also, why do we want to return a function vs. executing immediately?